### PR TITLE
add detail to screenshot view and add function to Screenshot factory to ...

### PR DESCRIPTION
**Pull request by rwuebker**
...get single screenshot

I added some code that Ruben and I were working on.  Not sure how to make the logout() function get called.

I have a feeling many groups will still be crunching stuff out on Monday, but if you guys wanna hit it on Sunday before the superbowl I can do that too..

### CODE REVIEW

**Comment by AndrewSouthpaw** [comment]
Rick -- please use HipChat for team communication. Leaving messages on PRs is not guaranteed to be seen by everyone. :)

File: client/app/src/js/controllers/screenshots.js [hunk 1]
```diff
@@ -13,6 +13,13 @@ function ScreenshotsController(screenshots, Screenshot) {
       console.log(data);
     });
   };
+  this.getScreenshot = function(id){
+    Screenshot.getScreenshot(id)
+    .success(function(data){
```
Comment (AndrewSouthpaw) [inline client/app/src/js/controllers/screenshots.js]: indent

File: client/app/src/js/services/Screenshot.js [hunk 2]
```diff
@@ -23,6 +23,14 @@ function Screenshot($http, User) {
     });
   };
 
+  var getScreenshot = function(id){
+    var user = User.getUser();
+    return $http.get('/api/user/' + user.username + '/screenshot/' + id)
+    .then(function(response){
```
Comment (AndrewSouthpaw) [inline client/app/src/js/services/Screenshot.js]: indent

**Comment by AndrewSouthpaw** [comment]
comments added. LMK when done and I'll merge.

**Comment by ruben1** [comment]
not sure what you mean by indent, but i don't see any problem with the indentation

which line specifically?

**Pull request merged**